### PR TITLE
feat(distribution): add functionality to send all pending inquiries

### PR DESCRIPTION
**Pull request by anehx**

### CODE REVIEW

File: packages/distribution/addon/components/distribution-navigation.js [hunk 1]
```diff
@@ -30,57 +25,24 @@ export default class DistributionNavigationComponent extends Component {
...
         return {
           ...inquiries,
-          [key]: uniqueByGroups(
-            objects.edges.map((edge) => ({
-              ...edge.node,
-              addressedGroupName: findGroupName(edge.node.addressedGroups),
-              controllingGroupName: findGroupName(edge.node.controllingGroups),
-            }))
-          ),
+          [key]: this.scheduler.groupCache.length
```
Comment (luytena) [inline packages/distribution/addon/components/distribution-navigation.js:32]: Add comment

**Review by luytena — CHANGES_REQUESTED** [review]

File: packages/distribution/addon/services/distribution.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                ...edge.node.addressedGroups,
+                ...edge.node.controllingGroups,
+              ]);
+            })
+            .flat(2)
+        ),
+      ];
+
+      this.scheduler.resolve(groupIds, "group");
+    });
```
[the remark this answers is not in the document]
Comment (anehx) [reply packages/distribution/addon/services/distribution.js]: Done

File: packages/distribution/addon/components/distribution-navigation.js (hunk 1 above)
Comment (anehx) [reply packages/distribution/addon/components/distribution-navigation.js:34]: Done

**Review by luytena — APPROVED** [review]

**Pull request merged**